### PR TITLE
Small tweaks to Build a Hotspot

### PR DESCRIPTION
**Pull request by joelachance**
I went through this today, and a couple of things I was finding:

1. On a Rasberry Pi, you can't copy all the code in a code cell if it's a multi-line cell, you'll get all of the added prompts. Removing these and adding some `&&` to run all cells in their entirety (on another note, I'm wondering if some less experienced might be confused by the `$` prompts?)

2. Adds a `cd ..` on LOC 126 for those who might get stuck.

3. I wasn't finding logs in `var/data/`, but i was finding the logs in `var/log/miner/...` Fixing those.

4. Removes whitespace (I know, I know)

Thanks for a good tutorial!

### CODE REVIEW

File: hotspot/developer-setup.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ In this guide, we are going to do something a little different. We are going to
 
 ![Build Your Own Hotspot](../.gitbook/assets/tutorial.png)
 
-And rest assured, while this guide uses Amazon AMI images to makes Miner deployment more or less plug-and-play, we are working on a Docker container version which will make it easy to not only run other Cloud Service providers, but your own server hardware or perhaps even your own gateway if the hardware is sufficient.  
+And rest assured, while this guide uses Amazon AMI images to makes Miner deployment more or less plug-and-play, we are working on a Docker container version which will make it easy to not only run other Cloud Service providers, but your own server hardware or perhaps even your own gateway if the hardware is sufficient.
```
Comment (joelachance) [inline hotspot/developer-setup.md:21]: just whitespace, no text changes

**Comment by lthiery** [comment]
@joelachance 
Thanks for the feedback. Merges with gitbook don't always work as anticipated and I already made some big updates to this article today, so I just cherry-picked the easy fixes.

I'm not fully aligned with the multi-line commands as I would like people to paste them in one by one to see where it fails if it does. I agree its slightly inconvenient, but debugging big && concatenations can get thorny.

**Comment by joelachance** [comment]
@lthiery good points, and sounds good! 
i think the main things I'd push for (if they didn't make it) are the `cd ..` on LOC 126 and checking on the log location. I followed this to the T and wasn't finding the logs in `var/data/log/`.

I'll close this PR, thx!

**Comment by lthiery** [comment]
Yep I absolutely absorbed those `/var/data/log` fixes - thank you for pointing those out! And I just added the `cd ...`
Thank you again for spending the time on this! It's really appreciated

**Pull request closed without merging**